### PR TITLE
udpating to include SJ spring run modificaitons

### PR DESCRIPTION
**Pull request by MaddeeRubenson**
adding specific logic for San Joaquin

### CODE REVIEW

File: data-raw/migratory_corridor/migratory_temps.R [hunk 1]
```diff
@@ -69,7 +69,7 @@ sjt_median_p20 <- sjt %>%
   summarise(p20 = sum(over20)/n()) %>%
   group_by(month) %>%
   summarise(median_p20 = median(p20)) %>%
-  mutate(group = 'San Joaquin')
```
Comment (ErinCain) [inline data-raw/migratory_corridor/migratory_temps.R]: Please remove the view statement.

File: data-raw/migratory_corridor/migratory_temps.R [hunk 2]
```diff
@@ -132,15 +132,19 @@ bind_rows(sdt_median_p20, sjt_median_p20, sact_median_p20) %>%
 
 median_p20 <- bind_rows(sdt_median_p20, sjt_median_p20, sact_median_p20)
 
-watershed_groups <- cvpiaData::watershed_ordering %>%
+watersheds_order <-  DSMhabitat::watershed_species_present %>%
```
Comment (ErinCain) [inline data-raw/migratory_corridor/migratory_temps.R]: Curious why you changed this?

File: data-raw/migratory_corridor/migratory_temps.R [hunk 3]
```diff
@@ -154,6 +158,36 @@ migratory_temperature_proportion_over_20 <- median_p20 %>%
...
+  select(order, watershed, month, median_p20) %>%
+  bind_rows(other_sheds) %>%
+  arrange(month, order) %>%
+  spread(month, median_p20) %>%
+  select(`1`:`12`) %>%
+  as.matrix()
+dimnames(migratory_temperature_proportion_over_20_sr) <- list(watersheds_order$watershed, month.abb)
+
+# check - should only be different for San Joaquin:
+migratory_temperature_proportion_over_20_sr == DSMtemperature::migratory_temperature_proportion_over_20
```
Comment (ErinCain) [inline data-raw/migratory_corridor/migratory_temps.R:169]: I think you should just update the number for SJ not recreate the whole data object. That way we do not have duplicative code that we have to update in 2 places

File: data-raw/migratory_corridor/migratory_temps.R (hunk 2 above)
Comment (ErinCain) [reply data-raw/migratory_corridor/migratory_temps.R]: If you want to change from cvpiaData::watershed_ordering, please use DSMflow::watershed_ordering not a DSMhabitat package

**Review by ErinCain — COMMENTED** [review]
deegree days logic looks fine but the migratory temp stuff is a little funky. Please try and simplify

File: data-raw/migratory_corridor/migratory_temps.R (hunk 2 above)
Comment (MaddeeRubenson) [reply data-raw/migratory_corridor/migratory_temps.R]: it doesn't seem like the cvpiaData library is used anymore...? I certainly don't have it. I changed to sourcing from DSMflow.

File: data-raw/migratory_corridor/migratory_temps.R (hunk 3 above)
Comment (MaddeeRubenson) [reply data-raw/migratory_corridor/migratory_temps.R:169]: updated to just update San Joaquin to match Tuolumne River migratory which is consistent with the previous values

Comment (ErinCain) [reply data-raw/migratory_corridor/migratory_temps.R:169]: I am not seeing this logic...still looks like you rebuild migratory temperature over 20

File: data-raw/migratory_corridor/migratory_temps.R [hunk 4]
```diff
@@ -154,6 +156,16 @@ migratory_temperature_proportion_over_20 <- median_p20 %>%
...
   as.matrix()
-dimnames(migratory_temperature_proportion_over_20) <- list(cvpiaFlow::watershed_ordering$watershed, month.abb)
+dimnames(migratory_temperature_proportion_over_20) <- list(watersheds_order$watershed, month.abb)
 
 usethis::use_data(migratory_temperature_proportion_over_20, overwrite = TRUE)
+
+# migratory for springRun
+## update to match Tuolumne River (30)
+migratory_temperature_proportion_over_20_sr <- migratory_temperature_proportion_over_20
+migratory_temperature_proportion_over_20_sr[31, ] <- migratory_temperature_proportion_over_20_sr[30, ]
```
Comment (MaddeeRubenson) [inline data-raw/migratory_corridor/migratory_temps.R:166]: @ErinCain this is the updated logic. 

`migratory_temperature_proportion_over_20` gets built and then I replace `San Joaquin River` with values from `Tuolumne River`. This logic was vetted by previously running `migratory_temperature_proportion_over_20` without SJ in `missing`

**Pull request merged**